package.json

```diff
@@ -73,6 +73,7 @@
     "rimraf": "^2.6.0",
     "sass-loader": "^7.0.1",
     "selenium-server": "^3.0.1",
+    "semantic-ui-css": "^2.3.1",
```

Comment:
Why are you adding multiple CSS libraries?

Reply:
I need a searchable dropdown with icons attached to each items.
Current dropdown llibrary works like charm, but it doesn't support icon images.
I'm experimenting Semantic UI for now.
I'll decide what UI library I'll use soon.